utils/mathUtils.js

```diff
@@ -0,0 +1,3 @@
+export const formatCommaSeparators = (number) => {
+  return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+};
```

Comment:
nice work on this little utility to add comma separators in numbers @RCode-Blue ! 😍